src/utils/InputHandler.js

```diff
@@ -0,0 +1,15 @@
+import { Console } from '@woowacourse/mission-utils';
+
+class InputHandler {
+    static async getInput() {
+        const input = await Console.readLineAsync('덧셈할 문자열을 입력해 주세요.\n');
+        return this.preprocessInput(input);
+    }
+
+    static preprocessInput(input) {
+        // '\\n'을 '\n'으로 변환
+        return input.replace('\\n', '\n').trim();
+    }
```

Comment:
클래스 내부에서만 사용되는 메서드라면 `#` prefix를 추가해서 private으로 지정해주는 것이 좋을 것 같습니다 😀
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Classes/Private_properties

추가로, `\\n`을 `\n`으로 변환해주신 이유가 궁금합니다!! 파싱을 간단하게 하기 위해서인가요..?!

```suggestion
    static #preprocessInput(input) {
        // '\\n'을 '\n'으로 변환
        return input.replace('\\n', '\n').trim();
    }
```

Reply:
`readLineAsync()`로 입력을 받을 때 "\n" 문자열이 "\\n"으로 입력되는 문제가 있었습니다.
그래서 올바른 파싱을 위해 입력된 "\\n"을 "\n"으로 변환해주는 전처리 과정을 추가했습니다.